note_importer.py

```diff
@@ -50,6 +52,13 @@ def copy_media_files(new_note: Note, other_note: Note) -> None:
             new_note.fields = [field.replace(file.name, new_filename) for field in new_note.fields]
 
 
+def remove_media_files(new_note: Note) -> None:
+    for file in files_in_note(new_note):
+        # As there still could be a card in the collection with this file, it shouldn't be deleted if that's the case
+        if not mw.col.find_cards(file.name):
+            new_note.col.media.trash_files([file.name])
+
+
```

Comment:
I think you can remove this because the user can always check media by running "Tools" > "Check media". Also, vanilla Anki doesn't check media after a note is added.

Reply:
I added this as the note is, in this case, not added. This would result in unused media just from clicking the edit button which would also be unwanted behaviour and may lead to confusion. Either way it is much more comfortable